state-sync/storage-service/server/src/optimistic_fetch.rs

```diff
@@ -248,7 +251,7 @@ async fn handle_ready_optimistic_fetches<T: StorageReaderInterface>(
                     let optimistic_fetch_request = optimistic_fetch.request.clone();
 
                     // Notify the peer of the new data
-                    if let Err(error) = notify_peer_of_new_data(
+                    if let Err(error) = utils::notify_peer_of_new_data(
```

Comment:
why move this though if it is only used here?

Reply:
Aah, this is used in a future PR, e.g., see here: https://github.com/aptos-labs/aptos-core/pull/9104 😄